docs/cedarling/cedarling-authz.md

```diff
@@ -9,49 +9,49 @@ tags:
-domain may use the `role` user claim of the OpenID Userinfo token; another domain may use the 
+domain may use the `role` user claim of the OpenID Userinfo token; another domain may use the
 `memberOf` claim in the OIDC id_token.
 
-Developers can also express a variety of policies beyond the limitations of RBAC by expressing ABAC 
-conditions, or combining ABAC and RBAC conditions. For example, a policy like Admins can access a 
-"private" Resource from the private network, during business hours. In this case "Admins" is the role, 
-but the other conditions are ABAC. Policy evaluation is fast because Cedar uses the RBAC role to 
+Developers can also express a variety of policies beyond the limitations of RBAC by expressing ABAC
+conditions, or combining ABAC and RBAC conditions. For example, a policy like Admins can access a
+"private" Resource from the private network, during business hours. In this case "Admins" is the role,
+but the other conditions are ABAC. Policy evaluation is fast because Cedar uses the RBAC role to
 "slice" the data, minimizing the number of entries on which to evaluate the ABAC conditions.
 
-The OIDC id_token JWT represents a Person authentication event. The access token JWT represents a 
-Workload authentication event. These tokens contain other interesting contextual data. The id_token 
-tells you who authenticated, when they authenticated, how they authenticatated, and optionally other 
-claims like the User's roles. An OAuth access token can tell you information about the Workload that 
-obtained the JWT, its extent of access as defined by the OAuth Authorization Server (*i.e.* the 
+The OIDC id_token JWT represents a Person authentication event. The access token JWT represents a
+Workload authentication event. These tokens contain other interesting contextual data. The id_token
+tells you who authenticated, when they authenticated, how they authenticatated, and optionally other
+claims like the User's roles. An OAuth access token can tell you information about the Workload that
+obtained the JWT, its extent of access as defined by the OAuth Authorization Server (*i.e.* the
 values of the `scope` claim), or other claims--domains frequently enhance the access token to
 contain business specific data needed for policy evaluation.
 
-The Cedarling authorizes a Person using a certain piece of software, which is called a "Workload". 
-From a logical perspective, `person_allowed AND workload_allowed` must be `True`. The JWT's, 
-Action, Resource and Context is sent by the application in the authorization request. For example, 
+The Cedarling authorizes a Person using a certain piece of software, which is called a "Workload".
+From a logical perspective, `person_allowed AND workload_allowed` must be `True`. The JWT's,
+Action, Resource and Context is sent by the application in the authorization request. For example,
 this is a sample request from a hypothetical application:
 
 ```
```

Comment:
we could add some syntax highlighting here so it's more obvious that this is Javascript code
```suggestion
```js
```

Reply:
resolved